crates/aptos/src/node/mod.rs

```diff
@@ -486,3 +494,174 @@ impl CliCommand<serde_json::Value> for ShowValidatorSet {
+    pub account_file: Option<PathBuf>,
+}
+
+#[async_trait]
+impl CliCommand<String> for MintCoins {
+    fn command_name(&self) -> &'static str {
+        "MintCoins"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<String> {
+        let mint_account_address = self.mint_account_address.unwrap_or_else(aptos_root_address);
+        let mint_key = if let Some(ref key) = self.mint_key {
+            key.private_key()
+        } else {
+            EncodingType::BCS
+                .load_key::<Ed25519PrivateKey>("mint key", Path::new(&self.mint_key_file_path))
+                .unwrap()
+        };
+        let faucet_account = LocalAccount::new(mint_account_address, mint_key, 0);
+        let service = Service::new(self.server_url, self.chain_id, faucet_account, None);
+
+        let accounts: HashSet<AccountAddress> = if let Some(accounts) = self.accounts {
+            accounts
+                .trim()
+                .split(',')
+                .map(|str| load_account_arg(str).unwrap())
+                .collect()
+        } else if let Some(path) = self.account_file {
+            let strings: Vec<String> =
+                serde_yaml::from_str(&std::fs::read_to_string(path.as_path()).unwrap()).unwrap();
+            strings
+                .into_iter()
+                .map(|str| load_account_arg(&str).unwrap())
+                .collect()
+        } else {
+            panic!("Either --accounts or --account-file must be specified");
+        };
+
+        let mut successes = vec![];
+        let mut failures = vec![];
+
+        // Iterate through accounts to mint the tokens
+        for account in accounts {
+            let response = mint::process(
+                &service,
+                MintParams {
+                    amount: self.amount,
+                    auth_key: None,
+                    address: Some(account.to_hex_literal()),
+                    pub_key: None,
+                    return_txns: None,
+                },
+            )
+            .await;
+            match response {
+                Ok(_) => successes.push(account),
+                Err(response) => {
+                    println!(
+                        "FAILURE: Account: {} Response: {:?}",
+                        account.to_hex_literal(),
+                        response
+                    );
+                    failures.push(account);
+                }
+            }
+        }
+
+        Ok(format!(
+            "Successes: {:?} Failures: {:?}",
+            successes, failures
+        ))
+    }
+}
+
+/// Show validator details of the validator set
+#[derive(Parser)]
+pub struct RunTestNode {
+    /// An overridable config for the test node
+    #[clap(long, parse(from_os_str))]
+    config_path: Option<PathBuf>,
+    /// The directory to save all files for the node
+    #[clap(long, parse(from_os_str), default_value = "/opt/aptos")]
+    node_dir: PathBuf,
+    /// Randomize ports rather than using defaults of 8080
```

Comment:
Can we also give them the mutually exclusive option to choose the port?

Reply:
Also "ports" implies you're randomizing other ports, but it looks like it is just the API port. If that's not true, mention as such in the help message, since you only mention the API port.